examples/tv-app/linux/args.gni

```diff
@@ -23,3 +23,5 @@ chip_system_project_config_include = "<SystemProjectConfig.h>"
 chip_project_config_include_dirs =
     [ "${chip_root}/examples/tv-app/tv-common/include" ]
 chip_project_config_include_dirs += [ "${chip_root}/config/standalone" ]
+
+chip_build_libshell = true
```

Comment:
this change seems to not be related to BLE layer changes

Reply:
both of these relate to the tv-app <-> tv-commissioning-app testing for TE6 so the thinking was to include both in a single PR. I can separate if you prefer, LMK